ddp-workspace/projects/ddp-sdk/src/lib/services/address.service.ts

```diff
@@ -19,10 +20,10 @@ export class AddressService extends UserServiceAgent<Address> {
         super(session, configuration, http, logger);
     }
 
-    public verifyAddress(address: Address): Observable<Address> {
-        return this.postObservable('/profile/address/verify', address, null, true).pipe(
+    public verifyAddress(address: Address): Observable<AddressVerificationResponse> {
+        return this.postObservable('/profile/address/verify', {...address, studyGuid: this.configuration.studyGuid}, null, true).pipe(
             map((data: any) => {
-                return new Address(data.body);
+                return new AddressVerificationResponse(data.body);
             }),
             catchError((error) => {
```

Comment:
New API call looks for study guid so server can look for study specific rules. In testboston, we look for rules that might trigger zip code warnings. If your zip code not in list, we will warn you.
Response is now a little more complicated. New type needed.